src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/resource/RecordARecallController.kt

```diff
@@ -21,7 +21,7 @@ class RecordARecallController(
   private val recordARecallService: RecordARecallService,
 ) {
   @PostMapping(value = ["/record-a-recall/{prisonerId}"])
-  @PreAuthorize("hasAnyRole('RECORD_A_RECALL', 'ROLE_CALCULATE_RELEASE_DATES__RECALL__CALCULATE__RW')")
+  @PreAuthorize("hasAnyRole('RECORD_A_RECALL', 'CALCULATE_RELEASE_DATES__RECALL__CALCULATE__RW')")
```

Comment:
you can prefix with ROLE_ (or not), the convention in CRD seems to be without (although it's mixed)

RECORD_A_RECALL role will be eventually removed from here